test/integration/dredd-transactions-test.coffee

```diff
@@ -5,7 +5,7 @@ proxyquire = require('proxyquire').noPreserveCache()
 fixtures = require('../fixtures')
 {assert} = require('../utils')
 createCompilationResultSchema = require('../schemas/compilation-result')
-dreddTransactions = require('../../src/dredd-transactions')
+dreddTransactions = require('../../src')
```

Comment:
I think that it would be more readable if you'd be explicit here - `dreddTransactions = require('../../src/index')`. Same goes for every occurrence.

Reply:
Will address. Do you mean just

`/src` ➡️  `/src/index`

or also

`/src/compile-uri` ➡️  `/src/compile-uri/index`

?

Reply:
All of them.

Reply:
😢 ok

Reply:
_discussed in person and the package-like imports have mercy now_